lib/requestWithOAuth2.js

```diff
@@ -63,15 +63,19 @@ function doOAuth(oauthUrl, clientId, clientSecret, username, password, callback)
   }, callback);
 }
 
-function doRequestWithAccessToken(url, accessToken, callback) {
-  var authHeader = 'Bearer ' + accessToken;
-
-  request.get({
-    url: url,
-    headers: {
-      Authorization: authHeader
+function doRequestWithAccessToken(requestGetArgs, accessToken, callback) {
+  var authHeader = {Authorization: 'Bearer ' + accessToken};
```

Comment:
This is just the authHeader. This is then added onto any other headers that were already passed in (part of requestGetArgs)